src/group_operations/addition_operation.jl

```diff
@@ -68,7 +68,7 @@ which simplifies for [`AdditionGroupOperation`](@ref) to ``Dι_{$(_math(:G))}(g)
 """
 
 @doc "$(_doc_diff_inv_add)"
-diff_inv(G::LieGroup{𝔽,AdditionGroupOperation}, g, X) where {𝔽} = -X
+diff_inv(G::LieGroup{𝔽,AdditionGroupOperation}, g, X) where {𝔽}
```

Comment:
Why did you remove the implementation?

Reply:
I do not fully remember – this PR is soooo long by now, that I might have missed a few details along the way already. I also spent most of the last 2 weeks just working on the humungous test coverage – that also lead to me forgetting some of the details before.

I think that default implementation breaks some tests?

Reply:
Ah I think its ambiguities. Without that definition, this line is just  attaching a doc string without doing a function definition.

Feel free to fight ambiguities there, but I feel I did nothing else the last 6 weeks-or-so than fighting ambiguities and/or trying to get test coverage done. So after a few hours I am sometimes just tired and if all tests work without the “fancy generic default”, and with that fancy default I am having several special cases to define anyways – I by now (after 2 months of just working on this code-wise) get a bit tired.

Reply:
I will check that then.

Reply:
And sure – fancy defaults are super nice to have – but for me there is a tradeoff in time-consuming fights as well. I would prefer this PR to be of finite length at some point (that I do not see yet).

Reply:
> I will check that then.

Good luck. I think I tried for a whole weekend or so.

Reply:
I've just added it back and it just works.